_includes/exhibits.js

```diff
@@ -7,13 +7,11 @@ const flatten = function(items) {
 const round1 = function(n) {
   return Math.round(n * 10) / 10;
 };
+
 const round4 = function(n) {
   const N = Math.pow(10, 4);
   return Math.round(n * N) / N;
 };
-const show5 = function(n) {
-  return ('' + n).slice(0, 5);
-};
```

Comment:
Unused function for rounding integers. Last used in old editor to show coordinates.